src/containers/DAO/pages/DAOCreate/DAOCreate.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
```

Comment:
Can we change this file to `CreateProposal.tsx`, as the page is for creating a proposal